setup.py

```diff
@@ -78,6 +78,14 @@
 package_info['package_data'].setdefault(PACKAGENAME, [])
 package_info['package_data'][PACKAGENAME].append('data/*')
 
+# Adam's note: I don't know what to do with this.
+# # Define entry points for command-line scripts
+# entry_points = {}
+# entry_points['console_scripts'] = [
+#     'astropy-package-template-example = packagename.example_mod:main',
+# ]
```

Comment:
@astrofrog: can you tell me what ought to be done with this?  Should `packagename` be changed to `astroquery` here?  Should `example_mod` also be astroquery?  I'm unfamiliar with this.  Maybe this is only for command line scripts, which astroquery currently does not include?